.zprofile

```diff
@@ -0,0 +1 @@
+export PATH=$PATH:/Users/^YourAccount^/^YourPath^/flutter/bin
```

Comment:
What is the use of this file?

Reply:
Oh yes this might be added while I was trying to add flutter to zsh lol :(((((

Reply:
Remove this file :-)